### DISCUSSION TITLE
ImageMagick refused to convert the animated GIF due to CPU and to tmpfs

### DISCUSSION THREAD

**Discussion by gusbemacbe** [ImageMagick/ImageMagick#2436] [Help]
I tried with these solutions below, but unsuccessfully:

- [ImageMagick convert problem, command killed or silently exits](https://askubuntu.com/questions/576717/imagemagick-convert-problem-command-killed-or-silently-exits)
- [Using disk for processing of huge images](https://www.imagemagick.org/discourse-server/viewtopic.php?t=33434)
- [ImageMagick convert out of memory](https://stackoverflow.com/questions/48423113/imagemagick-convert-out-of-memory)
- [ImageMagick using more than 2GB of memory to convert PDF files](https://stackoverflow.com/questions/15384879/imagemagick-using-more-than-2gb-of-memory-to-convert-pdf-files)

My animated GIF has 13.5MB. My computer has 16GB RAM memory. I have 8 CPUs of AMD Ryzen 7 2700X. I received the error:
Only less than 6% of my SSD is busy. I do not have a `/tmp` partition, it is already integrated to the `/` root partition. My RAM memory did not get full. 

```
$ LANG=C convert -limit memory 2MB -limit map 2MB -limit area 1000MB -delay 2 -loop 0 "Bug do pagamento.gif" Extraídos/bug.png
convert: unable to write pixel cache '/tmp/magick-RlftdgBD6MZCwArQzkuz5QI-kJtui7yk': No space left on device @ error/cache.c/WritePixelCachePixels/5830.
convert: corrupt image `Bug do pagamento.gif' @ error/gif.c/DecodeImage/505.
convert: corrupt image `Bug do pagamento.gif' @ error/gif.c/ReadGIFImage/1368.
convert: no images defined `Extraídos/bug.png' @ error/convert.c/ConvertImageCommand/3285.
zsh: exit 1     LANG=C convert -limit memory 2MB -limit map 2MB -limit area 1000MB -delay 2  
LANG=C convert -limit memory 2MB -limit map 2MB -limit area 1000MB -delay 2    2,84s user 2,60s system 99% cpu 5,454 total
```

- `tmpfs`:

```
LANG=C df -h /tmp
Filesystem      Size  Used Avail Use% Mounted on
tmpfs           7,9G  7,2M  7,9G   1% /tmp
```

I also have already tried with the alternative command `-limit memory 4GB`, but the same error.

**Comment by snibgo** [comment]
> convert: corrupt image `Bug do pagamento.gif' @ error/gif.c/DecodeImage/505.

It seems your GIF is corrupt. I suggest you paste a link to it, so it can be tested.
